common/src/main/java/whocraft/tardis_refined/common/capability/TardisLevelOperator.java

```diff
@@ -45,13 +46,19 @@ public class TardisLevelOperator {
     private final TardisInteriorManager interiorManager;
     private final TardisPilotingManager pilotingManager;
     private final TardisWaypointManager tardisWaypointManager;
-
     private final FlightDanceManager flightDanceManager;
     private final TardisClientData tardisClientData;
     private final UpgradeHandler upgradeHandler;
     private final AestheticHandler aestheticHandler;
 
 
+    private int tardisState = 0;
```

Comment:
Needs documentation to clarify this variable's purpose, its available values and the meaning of those values.